core/src/main/java/bisq/core/filter/FilterManager.java

```diff
@@ -398,6 +398,12 @@ public boolean isNodeAddressBanned(NodeAddress nodeAddress) {
                         .anyMatch(e -> e.equals(nodeAddress.getFullAddress()));
     }
 
+    public boolean isAutoConfExplorerBanned(String address) {
+        return getDevFilter() != null &&
```

Comment:
Here we need to use `getFilter` not `getDevFilter`. DevFilter is the locally stored of the filter admin. Not great naming... should be changed to a more clear name...

Reply:
Argh... thanks for finding that, switched to devFilter for testing: looks like I forgot to revert.